### PR TITLE
fix(directory_user): custom idp in test must be postfixed with -platform

### PR DESCRIPTION
**Pull request by v0lkc**
## Purpose

While working on a different issue, I figured out that the tests for directory_user are wrongly setup. The IDP must be postfixed with `-platform` in order to retrieve valid values. 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* check the service response in the updated fixtures

## What to Check

Verify that the following are valid

* ...

### CODE REVIEW

File: internal/provider/datasource_directory_user_test.go [hunk 1]
```diff
@@ -47,15 +47,15 @@ func TestDataSourceDirectoryUser(t *testing.T) {
...
 						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "user_name", "jenny.doe@test.com"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "origin", "terraformint"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "active", "false"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "family_name", ""),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "given_name", ""),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "id", ""),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "origin", "terraformint-platform"),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "active", "true"),
+                        resource.TestCheckResourceAttr("data.btp_directory_user.uut", "family_name", "unknown"), //FIXME should be empty, see NGPBUG-357810
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "given_name", "unknown"), //FIXME should be empty, see NGPBUG-357810
```
Comment (v0lkc) [inline internal/provider/datasource_directory_user_test.go:57]: As you can see this is inconsistent to the response for the default IDP. Here the service returns `unknown` as string, in the other case it returns `null`. I've opened a follow up issue in our internal issue tracking system (see NGPBUG-357810).

File: internal/provider/datasource_directory_user_test.go [hunk 2]
```diff
@@ -47,15 +47,15 @@ func TestDataSourceDirectoryUser(t *testing.T) {
...
 						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "user_name", "jenny.doe@test.com"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "origin", "terraformint"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "active", "false"),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "family_name", ""),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "given_name", ""),
-						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "id", ""),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "origin", "terraformint-platform"),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "active", "true"),
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "family_name", "unknown"), //FIXME should be empty, see NGPBUG-357810
+						resource.TestCheckResourceAttr("data.btp_directory_user.uut", "given_name", "unknown"),  //FIXME should be empty, see NGPBUG-357810
```
Comment (v0lkc) [inline internal/provider/datasource_directory_user_test.go:57]: As you can see this is inconsistent to the response for the default IDP. Here the service returns unknown as string, in the other case it returns null. I've opened a follow up issue in our internal issue tracking system (see [NGPBUG-357810](https://jira.tools.sap/browse/NGPBUG-357810)).

**Review by lechnerc77 — APPROVED** [review]
LGTM

File: internal/provider/datasource_directory_user_test.go (hunk 2 above)
Comment (lechnerc77) [reply internal/provider/datasource_directory_user_test.go:57]: As discussed

**Pull request merged**